### ISSUE TITLE
ngcc failed to run on ngx-sharebuttons@8.0.5

### ISSUE THREAD

**Issue by eng-dibo** [MurhafSousli/ngx-sharebuttons#553]
installing `ngx-sharebuttons` in a freshly created anglular project causing the error
```
ngcc failed to run on ngx-sharebuttons@8.0.5
```
[reproduction](https://stackblitz.com/edit/angular-ivy-1islbm)

I modied the [official Demo](https://stackblitz.com/edit/ngx-sharebuttons-9lghwm) and added `tsconfig.json` file in the root of your project to enable `Ivy`, then it failed with the same error.

```
{
  
  "angularCompilerOptions": {
    "enableIvy": false,
     }
}
```

**Comment by MurhafSousli** [comment]
This is a duplicate of #523 , but thanks for bringing this up again, I totally forgot about it! closing this one...

**Comment by MurhafSousli** [comment]
Fixed in v8.1.0, however, I don't know why the stackblitz is not working with it when ivy is enabled

**Comment by MurhafSousli** [comment]
Fixed in v8.1.0, however, I don't know why the stackblitz is not working with it when ivy is enabled
